Xamarin.Forms.Core/AppThemeColor.cs

```diff
@@ -1,4 +0,0 @@
-﻿namespace Xamarin.Forms
-{
-	public class AppThemeColor : OnAppTheme<Color> { }
```

Comment:
I was going to turn this helper internal, but if it's internal, it's not needed at all